lint-configs/lint-requirements.txt

```
@@ -0,0 +1,4 @@
+# Lock to v18.x until we can upgrade our code to meet v19's formatting standards.
+clang-format~=18.1
+clang-tidy>=19.1.0
+yamllint>=1.35.1
```

```suggestion
yamllint>=1.35.1

```

We try to end all files with an empty line.